nixie/unittests/test_verbose.py

```
@@ -0,0 +1,77 @@
+    with caplog.at_level(logging.INFO, logger="nixie.cli"):
+        assert await render_block(block, tmp_path, cfg_path, path, 1, semaphore)
+
+    mmd = tmp_path / "doc_1.mmd"
+    svg = mmd.with_suffix(".svg")
+    expected = shlex.join(get_mmdc_cmd(mmd, svg, cfg_path))
+    assert expected in caplog.text
+
+
+@pytest.mark.asyncio
+async def test_render_block_silent_without_verbose(
+    monkeypatch, tmp_path: Path, caplog: pytest.LogCaptureFixture
```

Ruff (0.12.2)

Missing type annotation for function argument `monkeypatch`

(ANN001)